webpack/plugins.js

```diff
@@ -52,12 +57,8 @@ const prodPlugins = [
     ],
   }),
   new webpack.optimize.UglifyJsPlugin({
-    mangle: {
-      keep_fnames: true,
-    },
-    compress: {
-      warnings: false,
-    },
+    beautify: false,
+    comments: false,
```

Comment:
mangle fnames was broken in one of the angular RCs. But if it's fixed now let's add it back because it helps a lot.